skyportal/handlers/api/gcn.py

```diff
@@ -184,6 +184,10 @@ def get(self, dateobs=None):
               application/json:
                 schema: Error
         """
+
+        page_number = int(self.get_query_argument("pageNumber", 1))
+        n_per_page = int(self.get_query_argument("numPerPage", 100))
```

Comment:
Add try catch with self error in case inputs are not integers.

Reply:
done